testsuite/model/pom.xml

```diff
@@ -100,6 +100,11 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-model-map-ldap</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons</artifactId>
```

Comment:
Could you please elaborate on why we need this dependency? Looks like you added after the first round of reviews?

Reply:
@pedroigor : the earlier wrapper which was used was failing the testsuite compilation and tests.  So with current scenario I was trying to fix additional dependancies that were required. I realised that these should come in a separate PR. This approach created additional failures, so I am now using environment variable.